Source/WebCore/platform/graphics/transforms/TransformOperationsSharedPrimitivesPrefix.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 1999 Antti Koivisto (koivisto@kde.org)
+ * Copyright (C) 2004, 2005, 2006, 2007, 2008 Apple Inc. All rights reserved.
```

Comment:
Probably too much copyright for this one function; would be nice to get it more precise.